appjs/searchbar.js

```diff
@@ -353,6 +353,30 @@ function autocomplete(inp, arr) {
                     else if(inp.value.toUpperCase() == "EXCESS-3 CODE CONVERTOR"){
                         document.getElementById('ex3').click();
                     }      
+                    else if (inp.value.toUpperCase() == "LINEAR REGRESSION CALCULATOR") {
+                        $("#lrcs").slideToggle();    
+                    }
+                    else if(inp.value.toUpperCase() == "LINEAR REGRESSION CALCULATOR"){
+                        document.getElementById('lrcs').click();
+                    }      
+                    else if (inp.value.toUpperCase() == "COVARIANCE CALCULATOR") {
+                        $("#covs").slideToggle();    
+                    }
+                    else if(inp.value.toUpperCase() == "COVARIANCE CALCULATOR"){
+                        document.getElementById('covs').click();
+                    }      
+                    else if (inp.value.toUpperCase() == "SKEWNESS CALCULATOR") {
+                        $("#skews").slideToggle();    
+                    }
+                    else if(inp.value.toUpperCase() == "SKEWNESS CALCULATOR"){
+                        document.getElementById('skews').click();
+                    }      
+                    else if (inp.value.toUpperCase() == "SOLVE FOR X") {
+                        $("#slvxs").slideToggle();    
+                    }
+                    else if(inp.value.toUpperCase() == "SOLVE FOR X"){
+                        document.getElementById('slvxs').click();
+                    }      
```

Comment:
This seems the instructions are repeating, as I pointed out in the previous PR. Kindly fix it